rainbow/src/main/java/org/projectrainbow/commands/_CmdVer.java

```diff
@@ -40,12 +40,16 @@ public void execute(MinecraftServer minecraftServer, ICommandSender cs, String[]
         );
         _DiwUtils.reply(cs, "Checking for update, please wait...");
         String latest = Updater.getLatestVersion();
-        if (Bootstrap.buildNumber == latest) {
-            _DiwUtils.reply(cs, "You are running the latest version.");
+        if (!latest.startsWith("ERROR")) {
+            if (Bootstrap.buildNumber == latest) {
+                _DiwUtils.reply(cs, "You are running the latest version.");
+            } else {
+                int currentNum = Integer.valueOf(Bootstrap.buildNumber);
```

Comment:
You should definitely catch exceptions here. I think `Bootstrap.buildNumber` will be set to `"unknown"` when running a self-compiled build of Rainbow. Though it might also be `null` in some obscure cases.